gtsfm/bundle/bundle_adjustment.py

```diff
@@ -285,6 +289,48 @@ def __cameras_to_model(
 
         return sorted(list(cameras))
 
+    def run_ba_at_threshold(
```

Comment:
nit: not sure if 'threshold' is important enough to be in the name, since its optional.

Reply:
Updated the name now to `run_ba_stage_with_filtering()`.